porting/build_and_boot/Halium_install.rst

```diff
@@ -77,4 +77,7 @@ For **Halium 9.0 ports**, use this command::
 
 where [SYSTEM_IMAGE] will be the file ``android-rootfs.img`` you downloaded and extracted, or alternatively the file ``system.img`` you built yourself.
 
-The script will copy and extract the files to their proper places, then allow you to set the user password for your device (the *phablet* user's password).
\ No newline at end of file
+The script will copy and extract the files to their proper places, then allow you to set the user password for your device (the *phablet* user's password).
+
+.. Note::
+    halium-install might fail due to unconfigured bintfmt-support for qemu throwing errors such as 'chroot: failed to run command ‘passwd’: Exec format error'.    To fix this,/proc/sys/fs/binfmt_misc/register should be updated. `binfmt_manager <https://github.com/mikkeloscar/binfmt_manager>`_ by `@mikkeloscar <https://github.com/mikkeloscar/>`_ can provide quickfix.
```

Comment:
* the link doesn't open: https://ci.ubports.com/job/docs.ubports.com/job/PR-451/3/artifact/_build/html/porting/build_and_boot/Halium_install.html 
* Add a space after "Fix this,"
* enclose the path /proc/sys/fs/binfmt_misc/register into backticks like so: `/proc/sys/fs/binfmt_misc/register`

Reply:
done, done and done. Thanks for the review. I don't think it is a bug in halium-install. But, I'd rather it to indicate me that binfmt-support is not configured properly.